### PR TITLE
Updates to DP9 per PTD edits

### PR DESCRIPTION
**Pull request by tallardyce**
All edits made to #9 address security and privacy playbook principle

### CODE REVIEW

**Comment by tallardyce** [comment]
Paul, can you check the CI test results?  Seems to be failing on something I don't know about.

**Comment by pjackson28** [comment]
@tallardyce It looks like like there are Markdownlint errors (issues in the Markdown that could potentially cause issues). You can see all the errors from lines 789 to 883 in the automated build log at: https://travis-ci.org/canada-ca/digital-playbook-guide-numerique/builds/382702409. It gives a description of each error and the lines in the .md file where they occur.

Would you like me to clean up the issues for you or would you prefer to try to fix them yourself (with me providing guidance)? I'm happy to help either way.

**Comment by pjackson28** [comment]
@tallardyce For other pull requests, you can also see the details for why a check has failed by clicking the "Details" link to the right of the failed check. For this PR, that "Details" links points to the build log that I referenced in the previous comment.

**Comment by tallardyce** [comment]
I'll try and take care of it -- can't be anything too complex; there's nothing complicated in the md

**Comment by pjackson28** [comment]
@tallardyce It looks like the last build failure was because there was a network hiccup during the network validation. I restarted the build which should hopefully address the issue.

**Comment by pjackson28** [comment]
@tallardyce I'll help with this validation error, you added an internal link that needs to be added to the link validation exception list since external link validation will fail for any internal link. The link in question is: http://intranet.canada.ca/wg-tg/go-sg/sim-gsi/spin-amps/spin-amps-eng.asp

**Comment by tallardyce** [comment]
thanks!  Noticed the internal link issue and was about to remove it, so thanks for being on top of it.

**Comment by pjackson28** [comment]
No problem. I also added "(internal to Government of Canada)" to the link as well so users know that it is an internal link.

**Comment by pjackson28** [comment]
It passes the build so I'll quickly replicate your changes on the French version before merging in the PR.

I noticed you changed all list items from "-" to "*". Either is acceptable so I think the reason you got the Markdownlint issues was probably because "-" and "*" were mixed in the same list (so if it saw "*" on the previous list item then it would then expect to see "*" on the next list item). The reason it flags it as an error is that using a different list item character in the same list is supposed to trigger a new list in Markdown.

**Comment by pjackson28** [comment]
Just to check, should I go ahead with merging in the PR once the French changes are in place?

**Comment by tallardyce** [comment]
I used * for the list items.. what's the best practice?  excited the CI checks passed tho.  I'd like to give Po the final check on the content, but should be soon. @ptd-tbs

**Comment by pjackson28** [comment]
I don't think there really is a best practice, although "-" seems to be most common in the Playbook so far (no need to change it as it makes no real difference in the end).

I noticed that you removed the OneGC architectural checklist items. How will the Security-related items be integrated? Are you using newer language for those items? If so, we should apply the dpgn-onegc-architectural tag to them.

**Comment by pjackson28** [comment]
I ran a quick test and it looks like when you use either "-" or "*" for a list item then you need to use it for the remainder of the page. Markdownlint doesn't seem to care which you use but doesn't like switching between them within the same page (even in different lists).

**Comment by tallardyce** [comment]
Cool..good to know.  So fussy.

**Comment by pjackson28** [comment]
Replacement PR for #144 (creating a reference to the other PR).

File: en/9-address-security-privacy-risks.md [hunk 1]
```diff
@@ -68,69 +93,53 @@ altLangPage: 9-gerer-risques-matiere-securite-protection-renseignements-personne
...
-- document any outstanding legal concerns, such as data protection or data sharing **(Digital Service Standard (Ontario))**
-- document the privacy policy and rationale **(Digital Service Standard (Ontario))**
+* Ensure that a preliminary privacy impact assessment (PPIA) or privacy impact assessment (PIA) has been completed if personal information is involved in the service.
+* Determine, in consultation with a records officer, what data is collected and why, how it is used or shared, how it is stored and secured, and how long it is kept.
+* Ensure that digital solutions and open data initiatives comply with requirements regarding the collection and protection of personal information.
+* Where collecting personal information, inform users about privacy rights and protections, and about their right to access and correct their own personal information.
+* Use appropriate de-identification strategies to minimize the risk of disclosing personal information.
+* Incorporate privacy safeguards into partnership and data sharing agreements.
+* Develop your system with the 7 principles of Privacy by Design in mind.
```
Comment (thomasgohard) [inline en/9-address-security-privacy-risks.md]: This should link to the 7 principles of Privacy by Design.

File: en/9-address-security-privacy-risks.md [hunk 2]
```diff
@@ -1,22 +1,29 @@
...
 altLang: fr
 altLangPage: 9-gerer-risques-matiere-securite-protection-renseignements-personnels
 ---
 <div class="dpgn-section-intro-standard">
 
-**[TODO: Add/revise introductory text]**
+Canadians who use government services must have confidence that:
 
+* any information they provide is confidential and stored appropriately
+* the system they’re using is safe and secure
```
Comment (ptd-tbs) [inline en/9-address-security-privacy-risks.md]: Suggest moving as the last bullet

File: en/9-address-security-privacy-risks.md [hunk 3]
```diff
@@ -1,22 +1,29 @@
...
+* they know how their information will be used by government
+* they can easily retrieve information they provide
+* that their privacy is protected while they use the service, and afterwards
+
+If a service cannot guarantee confidentiality, integrity and availability of the system, people will not use it. Effective cyber and IT security is an essential enabler of digital transformation. Securing #GCDigital requires the delivery of government services that are safe, secure and trusted by Canadians.
 </div>
 
 <div class="dpgn-section-guidelines">
 
 **Guidelines:**
```
Comment (ptd-tbs) [inline en/9-address-security-privacy-risks.md]: Suggest removing this section.

File: en/9-address-security-privacy-risks.md (hunk 1 above)
Comment (ptd-tbs) [reply en/9-address-security-privacy-risks.md]: Recommend using this reference - https://www.ipc.on.ca/wp-content/uploads/Resources/7foundationalprinciples.pdf

File: en/9-address-security-privacy-risks.md (hunk 2 above)
Comment (pjackson28) [reply en/9-address-security-privacy-risks.md]: Moved to the end of the list.

File: en/9-address-security-privacy-risks.md (hunk 1 above)
Comment (pjackson28) [reply en/9-address-security-privacy-risks.md]: Turned into a link using the recommended reference.

**Comment by pjackson28** [comment]
I made the final changes requested for approval and then merged the content in. Any further changes can be addressed through a future PR.

One change that should be considered is how to properly integrate the OneGC architectural checklist/Build It Right items that relate to this standard. Previously there were items for Pan-Canadian Trust Framework and HTTPS only but the Pan-Canadian Trust Framework one was reworded and the HTTPS only was reworded and limited to just APIs. Suggest working with @tdandrea23 on the updated wording for those two items and how best to integrate them with Standard 9.

**Comment by ptd-tbs** [comment]
Thanks for your help on this Paul. We have provided feedback to @tdandrea23 on the checklist. Note that we have asked for HTTPS and PCTF to be updated so that it is more all encompassing:

Use Trusted Digital Identities - Incorporate widely accepted Trusted Digital IDs to effectively identify users and provide access across platforms and devices to services offered in multiple jurisdictions. Accommodate an ecosystem of identity management approaches to improve service accessibility and experience.

Make things secure - Integrate security from the outset. Design the system so that it can resist attacks. Apply the Assume Breach principle and perform threat modeling to identify and eliminate design issues.

**Pull request merged**